apps/chat-e2e/src/tests/messageTemplate.test.ts

```diff
@@ -0,0 +1,858 @@
+        await messageTemplateModal.getTemplateRowValue(1).click();
+        await messageTemplateModalAssertion.assertElementText(
+          messageTemplateModal.getFieldBottomMessage(
+            messageTemplateModal.getTemplateRowContent(1),
+          ),
+          ExpectedConstants.messageTemplateRequiredField,
+        );
+      },
+    );
+
+    await dialTest.step(
+      'Set correct value into message, clear the row fields and verify error appears',
+      async () => {
+        await messageTemplateModal.getTemplateRowContent(1).fill(request);
+
+        await messageTemplateModal.getTemplateRowContent(1).fill('');
+        await messageTemplateModal.getTemplateRowValue(1).fill('');
+
+        await messageTemplateModalAssertion.assertElementText(
+          messageTemplateModal.getFieldBottomMessage(
+            messageTemplateModal.getTemplateRowContent(1),
+          ),
+          ExpectedConstants.messageTemplateRequiredField,
+        );
+        await messageTemplateModalAssertion.assertElementText(
+          messageTemplateModal.getFieldBottomMessage(
+            messageTemplateModal.getTemplateRowValue(1),
+          ),
+          ExpectedConstants.messageTemplateRequiredField,
+        );
+      },
+    );
+
+    await dialTest.step(
+      'Set correct values into row fields and verify errors disappear',
+      async () => {
+        await messageTemplateModal.getTemplateRowContent(1).fill(request);
+        await messageTemplateModal
+          .getTemplateRowValue(1)
+          .fill(firstRowFirstVar);
+        await messageTemplateModalAssertion.assertElementState(
+          messageTemplateModal.getFieldBottomMessage(
+            messageTemplateModal.getTemplateRowContent(1),
+          ),
+          'hidden',
+        );
+        await messageTemplateModalAssertion.assertElementState(
+          messageTemplateModal.getFieldBottomMessage(
+            messageTemplateModal.getTemplateRowValue(1),
+          ),
+          'hidden',
+        );
+      },
+    );
+
+    await dialTest.step(
+      'Set the template that does not match the message and verify error is displayed',
+      async () => {
+        await messageTemplateModal
+          .getTemplateRowContent(1)
+          .fill(firstRowContent);
+        await messageTemplateModal
+          .getTemplateRowValue(1)
+          .fill(
+            firstRowTemplate.substring(
+              0,
+              firstRowTemplate.indexOf(firstRowSecondVar),
+            ),
+          );
+        await messageTemplateModalAssertion.assertElementText(
+          messageTemplateModal.getFieldBottomMessage(
+            messageTemplateModal.getTemplateRowValue(1),
+          ),
+          ExpectedConstants.messageTemplateMismatchTextErrorMessage,
+        );
+      },
+    );
+
+    await dialTest.step(
+      'Correct the template value and verify error disappears',
+      async () => {
+        await messageTemplateModal.getTemplateRowValue(1).fill(request);
```

Comment:
It just replaces one incorrect value with another even though the test step aim is to verify the disappearance of the error message

Reply:
fixed